geniza/corpus/solr_queryset.py

```diff
@@ -91,6 +91,16 @@ def _search_term_cleanup(self, search_term):
         # to avoid it being interpreted as a boolean
         search_term = self.re_shelfmark_nonbool.sub("BL or", search_term)
 
+        # regex to match terms in doublequotes, but not following a colon, at the
+        # beginning of the string or after a space. (non-greedy to prevent matching
+        # the entire string if there are multiple sets of doublequotes)
+        search_term = re.sub(
+            r'\B(?<!:)(".+?")',
```

Comment:
I think you should also exclude fuzzy and proximity searches, which could be used with quotes. I think adding a word boundary after the end quote should be sufficient, but would be good to add test cases.

Reply:
I wasn't able to get it working with just a word boundary, but either way they both use `~`, so I'll just match that explicitly.